packages/phrasebook/src/locale/zh-hant/password-field.ts

```diff
@@ -2,7 +2,8 @@ import { Phrasebook } from '../../translation.js';
 
 const translations = {
   SHOW_PASSWORD: '顯示密碼',
-  HIDE_PASSWORD: '隱藏密碼'
+  SHOW_PASSWORD_ON: '顯示密碼開啟，密碼可見', // machine translated
+  SHOW_PASSWORD_OFF: '顯示密碼關閉，密碼被隱藏' // machine translated
```

Comment:
I think we should contact translation team for translate this.

Reply:
exactly! Let's finalize our implementation first so the translation would be finalized as well.

Reply:
translation updated.